sources/europe/ua/Ukraine-Mykolaiv2018(mbk).geojson

```diff
@@ -0,0 +1,2569 @@
+{
+    "type": "Feature",
+    "properties": {
+        "id": "UkraineMykolaiv2018",
+        "name": "Ukraine - Mykolaiv 2018",
+        "type": "tms",
+        "url": "http://mbk.mkrada.gov.ua/map/rtile/carto_2013928517078615452/ua/{zoom}/{x}/{y}.png",
+        "category": "photo",
+        "permission_osm": "explicit",
```

Comment:
could you also include a license_url where more details about permission to use this data in OSM can be found?

Reply:
there is no specific indication of the license, well, there is a law of Ukraine that allows the use of open data in the public domain and use it for any purpose, including commercial

Reply:
@grischard another reason why we need that "license" field reinstated, so in this case it can be set to `PUBLIC_DOMAIN` to keep track of why this data can be used in OSM.